src/main/java/com/cloudbees/jenkins/support/SupportCommand.java

```diff
@@ -107,7 +98,7 @@ protected int run() throws Exception {
         }
 
         public OutputStream call() throws IOException {
-            File f = File.createTempFile(filename, ".zip");
+            File f = File.createTempFile(filename, "");
```

Comment:
This is wrong. Even if `filename` ends in `.zip`, you will end up with something like this:

```
$ jrunscript -e 'println(java.io.File.createTempFile("foo.zip", ""))'
/tmp/foo.zip6023275810588304403
```